src/client/java/teammates/client/scripts/StatisticsPerInstitute.java

```diff
@@ -254,13 +253,13 @@ private void print(List<InstituteStats> statList) {
     private List<InstituteStats> convertToList(
             HashMap<String, HashMap<Integer, HashSet<String>>> institutes) {
         List<InstituteStats> list = new ArrayList<>();
-        for (Map.Entry<String, HashMap<Integer, HashSet<String>>> entry : institutes.entrySet()) {
+        institutes.forEach((insStatName, inStatStudents) -> {
```

Comment:
Even though the object concerned afterwards is an `InstituteStats`, the map is really about institutes so it's more apt to name `insName` and `insStudents`.